st2actions/st2actions/runners/pythonrunner.py

```diff
@@ -77,6 +78,21 @@ def __init__(self, config=None):
     def run(self, **kwargs):
         pass
 
+    def _set_up_logger(self):
```

Comment:
This method is not being called anywhere. Are we missing some integration code?

Also, similar impl exists in pythonactionwrapper now so not a good idea to override here anyway. See https://github.com/StackStorm/st2/blob/master/st2actions/st2actions/runners/python_action_wrapper.py#L110